gcc/config/riscv/riscv-c.cc

```diff
@@ -265,7 +270,9 @@ riscv_pragma_intrinsic (cpp_reader *)
   const char *name = TREE_STRING_POINTER (x);
 
   if (strcmp (name, "vector") == 0
-      || strcmp (name, "xtheadvector") == 0)
+      || strcmp (name, "xtheadvector") == 0
+      || strcmp (name, "xsfvqmaccqoq") == 0
+      || strcmp (name, "xsfvqmaccdod") == 0)
```

Comment:
`#pragma riscv intrinsic "sifive_vector"`, so 
```suggestion
      || strcmp (name, "xtheadvector") == 0
      || strcmp (name, "sifive_vector"))
```